src/ResourceManagement/RecoveryServices/Microsoft.Azure.Management.RecoveryServices/project.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": "4.0.1-preview",
```

Comment:
@cormacpayne Moving to Autorest from Hyak has caused some breaking changes in the client, thus I've increased the major version.